README.md

```diff
@@ -1097,8 +401,6 @@ To prevent delays to release schedules, fuzzing is not restarted for a release i
 
 <hr>
 
-⚓  [Quick Start](#quick-start) • [Features](#features) • [Standards](#standards) • [API](#api) • [Options](#options) • [Usage](#usage) • [Fuzzing](#fuzzing-and-code-coverage) • [License](#license)
-
 ## Versions and API Changes
```

Comment:
Remove ", etc." in this sentence to make it less broad.  Maybe sentence isn't necessary?

> Bug fixes like detecting an error that was missed in prior version are excluded from SemVer as long as function parameters, etc. are unchanged.